Description: We don't need bundler
 Add dependencies in debian/control.
 Add require lines in test_helper.rb
Author: Praveen Arimbrathodiyil <praveen@debian.org>
Last-Update: 2014-01-14

--- a/tests/test_helper.rb
+++ b/tests/test_helper.rb
@@ -1,5 +1,14 @@
-require 'rubygems' if RUBY_VERSION < '1.9'
-require 'bundler/setup'
+# require 'rubygems' if RUBY_VERSION < '1.9'
+# require 'bundler/setup'
+
+# from gemspec
+require 'active_support'
+require 'eventmachine'
+require 'rake'
+require 'rdoc'
+require 'shindo'
+require 'sinatra'
+
 require 'excon'
 require 'delorean'
 require 'open4'
